tests/common/playground.rs

```diff
@@ -0,0 +1,123 @@
+use couchbase::{Cluster, CouchbaseResult, MutationResult};
+use lazy_static::lazy_static;
+use nu_test_support::playground::*;
+use serde::Serialize;
+use std::env;
+use std::path::PathBuf;
+use std::sync::{Arc, Mutex};
+
+lazy_static! {
+    static ref STATE: State = State {
+        connstr: env::var("CBSH_CONNSTRING").unwrap_or("localhost".to_string()),
+        bucket: env::var("CBSH_BUCKET").unwrap_or("default".to_string()),
+        scope: env::var("CBSH_SCOPE").unwrap_or("_default".to_string()),
+        collection: env::var("CBSH_COLLECTION").unwrap_or("_default".to_string()),
+        username: env::var("CBSH_USERNAME").unwrap_or("Administrator".to_string()),
+        password: env::var("CBSH_PASSWORD").unwrap_or("password".to_string()),
+    };
+    static ref CLUSTER: Mutex<Option<Arc<Cluster>>> = Mutex::new(None);
+}
+
+// dead_code seems to pick up several functions in this file even though they are used.
+#[allow(dead_code)]
+pub fn default_bucket() -> String {
+    STATE.bucket.clone()
+}
+
+#[allow(dead_code)]
+pub fn default_scope() -> String {
+    STATE.scope.clone()
+}
+
+#[allow(dead_code)]
+pub fn default_collection() -> String {
+    STATE.collection.clone()
+}
+
+struct State {
+    connstr: String,
+    bucket: String,
+    scope: String,
+    collection: String,
+    username: String,
+    password: String,
+}
+
+pub struct CBPlayground {}
```

Comment:
Once we need it we can add nu playground as a property of this. I did have it that way but dead_code complained at me.